### PR TITLE
Add Linux_Spanish CI job

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by JoeRobich** [comment]
@jmarolf please take a look

**Review by jmarolf — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -73,34 +73,35 @@ jobs:
...
-- job: Windows_Spanish
+- job: Linux_Spanish
   pool:
-   name: NetCorePublic-Pool
-   queue: buildpool.windows.10.amd64.es.vs2017.open
+    vmImage: 'ubuntu-latest'
   variables:
-    _os: Windows
+    _os: Linux
+    LC_ALL: es_ES.UTF-8
```
Comment (jmarolf) [inline azure-pipelines.yml:81]: honestly love this. want to enable it for roslyn-analyzers once all the tests run on linux

**Comment by RikkiGibson** [comment]
Is this something Roslyn should look at using?

**Comment by jmarolf** [comment]
@RikkiGibson I don't think this will provide the same test coverage. _All_ our tests run on linux whereas in roslyn only a subset do right?

**Pull request merged**